AUTHORS

```diff
@@ -0,0 +1,16 @@
+Ivo Nunes <ivo@elementaryos.org>

```

Comment:
We probably don't need this AUTHORS file since git keeps track of who has written which lines for us